wg-naming/recommendations/template.md

```diff
@@ -0,0 +1,17 @@
+# Recommendation: Old Term -> New Term
+
+**Status:** Accepted|Draft|Other
```

Comment:
What sort of workflow do we envision with this field?  I know KEPs have their own status field, but I feel like it's a source of confusion/toil, and would opt for the simplicity of "merged = accepted" if possible here.

If this is about driving implementation, I guess I was thinking KEPs would be used for that.

Reply:
I think that's fair. I'm going to keep the field in as `Accepted` for now and remove the other options, because I think it's important for the merged file to reflect that.